debian/io.elementary.installer-casper.install

```diff
@@ -0,0 +1 @@
+debian/99pop-installer usr/share/initramfs-tools/scripts/casper-bottom/
```

Comment:
Do we still want this to be named `pop-installer` or was this just a copy/paste thing?

Reply:
Looks like copy-paste since the file above is `debian/99elementary-installer`

```suggestion
debian/99elementary-installer usr/share/initramfs-tools/scripts/casper-bottom/
```